src/osd/OSD.cc

```diff
@@ -5560,15 +5560,17 @@ void OSD::flush_pg_stats()
 
 void OSD::send_beacon(const ceph::coarse_mono_clock::time_point& now)
 {
-  dout(20) << __func__ << dendl;
-  last_sent_beacon = now;
   const auto& monmap = monc->monmap;
   // send beacon to mon even if we are just connected, and the monmap is not
   // initialized yet by then.
-  if (monmap.epoch == 0 &&
+  if (monmap.epoch > 0 &&
```

Comment:
right, `||` was what was intended. maybe we can remove the comment above, which will be confusing then.